apps/build_stitched_index.cpp

```diff
@@ -343,7 +344,9 @@ int main(int argc, char **argv)
     path labels_file_to_use = final_index_path_prefix + "_label_formatted.txt";
     path labels_map_file = final_index_path_prefix + "_labels_map.txt";
 
-    convert_labels_string_to_int(label_data_path, labels_file_to_use, labels_map_file, universal_label);
+    std::set<std::string> raw_universal_label_set = {universal_label};
+    diskann::InMemFilterStore<uint32_t>::convert_labels_string_to_int(label_data_path, labels_file_to_use,
```

Comment:
I added this here because, the end user does not know if we have an internal mapping and how its done, similarly Index does not know that string labels is a thing, it just expects number labels. So filter store seems like an ideal place to do this conversion. That way it can also keep the mapping logic and also the map itself.